src/test/java/games/strategy/triplea/delegate/MatchesTest.java

```diff
@@ -31,18 +32,18 @@
 
 public final class MatchesTest {
 
-  private static final Match<Integer> IS_ZERO_MATCH = Match.of(it -> it == 0);
+  private static final Predicate<Integer> IS_ZERO_MATCH = Match.of(it -> it == 0);
   private static final Object VALUE = new Object();
 
-  private static <T> Matcher<Match<T>> matches(final @Nullable T value) {
-    return new TypeSafeDiagnosingMatcher<Match<T>>() {
+  private static <T> Matcher<Predicate<T>> matches(final @Nullable T value) {
```

Comment:
In the future, we should look to see if there is a similar existing Hamcrest matcher for `Predicate`.